integreat_cms/cms/templates/pages/page_form.html

```diff
@@ -123,24 +129,35 @@ <h1 class="heading">
                                         <i data-feather="x"></i>
                                     </span>
                                 {% endif %}
+                                <span>
+                                    <span class="fp fp-rounded fp-{{ other_language.primary_country_code }}"></span>
+                                    {% if other_language.secondary_country_code %}
+                                        <span class="fp fp-rounded fp-{{ other_language.secondary_country_code }}"></span>
+                                    {% endif %}
+                                </span>
                                 {{ other_language.translated_name }}
                             </a>
                         {% endif %}
                             </div>
                         </div>
                     </li>
                 {% endfor %}
-                {% if page_translation_form.instance.id %}
-                    <li class="self-start ml-auto px-2">
-                        {% trans 'Version' %}:
-                        {{ page_translation_form.instance.version }}
-                        (<a href="{% url 'page_revisions' page_id=page.id region_slug=region.slug language_slug=language.slug %}"
-                            class="text-blue-500 hover:underline">{% trans 'Show versions' %}</a>)
-                    </li>
+                {% if page %}
+                <div class="ml-auto z-10">
+                    {% include "content_language_switcher.html" with target='edit_page' content_id=page.id %}
+                </div>
                 {% endif %}
             </ul>
-            <div class="w-full mb-4 rounded border-2 border-blue-500 bg-white flex-auto">
+            <div class="w-full mb-4rounded border-2 border-blue-500 bg-white flex-auto">
```

Comment:
```suggestion
            <div class="w-full mb-4 rounded border-2 border-blue-500 bg-white flex-auto">
```
This is obviously a typo.